docs/config.md

```diff
@@ -52,6 +52,7 @@ All configuration files should be placed inside the application's configuration
 | `cover_img_width`                 | the width of the cover image (`image` feature only)                                      | `5`                                                     |
 | `cover_img_length`                | the length of the cover image (`image` feature only)                                     | `9`                                                     |
 | `cover_img_scale`                 | the scale of the cover image (`image` feature only)                                      | `1.0`                                                   |
+| `autoplay`                        | enable spotify autoplay                                                                  | `false`                                                   |
```

Comment:
this should a device config: https://github.com/aome510/spotify-player/blob/018d6c26ae1f7e4396fe43c6c353421595a929ed/docs/config.md#device-configurations